RFC-004-revising-licenses-page.md

```diff
@@ -0,0 +1,118 @@
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+## Proposal overview
+
+I believe we should rethink the licensing terms for website content, as listed on [thegooddocsproject.dev/licenses/](https://thegooddocsproject.dev/licenses/). As they're currently written, they provision everything as [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/) (a.ka. "CC BY 4.0), including all text, images, HTML code, and anything else.
+
+We may not always have the rights to relicense what we put on our page, and not every contributor is comfortable with having their info and likeness released as publicly usable.
+
+### Scope
+
+The scope of this is about the website's content. Licensing for our products (templates, doc tools, etc.) aren't part of this discussion, though this proposal does indicate where the primary home for those licenses should be.
+
+## Motivation
+
+For the upcoming blog, not everything we release will be ours. If we use stock photos for the purposes of social media optimization, we won't have the rights to relicense that content. We also may not have the rights to relicense other content we'll reuse, including any HTML/CSS/scripting to generate the pages.
+
+On a personal level, I don't release my likeness to public use, and the current website license terms makes me wary of continuing with the project. I used to work in games (and still do at times), and I've had too many death threats (honestly, one such threat is too many) and have seen too many friends be impersonated to want to give bad actions a contractual right to use my likeness however they like. (Bad actors will still use likenesses, but at least they can't defend their actions by citing a license that allows them to use one's info.)
+
+### Blocking
+
+[RFC002 (Procope/blog & video)](https://github.com/thegooddocsproject/request-for-comment/pull/3) is blocked by this proposal.
+
+## Proposal
+
+I'd like to shift the burden of licensing to the individual elements, rather than have a pure blanket. That means:
+
+- Our products (templates, doc tools, etc) have the BSD0 license in their readme or license text files, and not rely on the license website page for that notice
+- For blog posts etc, post pages would default to have "`This blog post's original text is (c) [year] [author name], and licensed under CC BY 4.0`". Currently, the blog is set up to allow for this default, with other changes allowed. Ryan and Bryan worked together to make sure this could be modular.
+- "Original text" covers not opening any quoted text (like in referencing a book's text) as being CC, and doesn't open any stock images from being CC. If a custom image is created by the author, we can add that to the CC license, which is part of why Ryan and Bryan worked to make the license field modular.
```

Comment:
issue (blocking): I'm concerned that this definition is getting too complicated. It is not a standard license term so we don't get the benefit of many lawyers having had reviewed it. It also means that we now force stakeholders to read the license terms. (One of the beauties of open source licenses is that you only need to read them once - not for every single project you get involved in.)
On first brush, I don't fully understand what it means, and I've read a lot of licenses and I care about what is written. A quick test is: Ask your employer's legal department if they have time to review this license for you. Ask if they would agree to the license terms, or whether they say "just use one of the standard open source licenses" (which is the usual response).

Reply:
How is it not a standard license term? It's a declaration of open content + license stated. Please show me what you mean by "non-standard"? That the declaration is more specific than "everything"?

Reply:
We can simplify the default blog license to "This blog post's text is..." And allow for a variant for "This blog post's text (aside from quoted text) is..."

That *slightly* complicates this, but if that makes the default license readable, and we have an easy way to set variants (which we currently do based on the work Bryan and I have already done), I'm more than fine with it.

Reply:
Comment (hopefully clarifying Ryan's previous comments): I also don't understand the nature of your objection, Cameron. Ryan is merely clarifying what "original text" refers to, as he explains in more detail in the section entitled "Some clarity" a few headings below this one in the RFC. As he states in that section:

> Note that you can specify with CC BY what content is covered under the license. It isn't inherent to the license that everything in a work is covered. You always say what's covered...

So all he's doing is following CC BY's standard and specifying what is covered in this context: the original text. That's the standard application of CC BY.

Reply:
I'll re-categorize my comment to:
thought (non-blocking): I have a mild feeling that this explanation is a bit long and legal, which could be annoying to a reader. (As is the case with almost all legal text.)
I'm fine with stepping back from my comment and deferring to advice from Ryan and Alyssa.

Reply:
I'll resolve this as a non-issue. These aren't for readers, they're for people explicitly duplicating and republishing our content elsewhere. Because (to restate in yet more words) most people "using" a blog are reading it, putting it in their minds, executing on whatever thoughts they get from it, and perhaps share it around. Rare is the instance where someone's use is about _copying_ and _republishing_ the content—and those people absolutely shouldn't be "annoyed" by legal text, or they're in the wrong line of work.

(This is also why Google's guidelines for publications is different from its open source patching guide.)